### PR TITLE
Project In Progress State

### PR DESCRIPTION
**Pull request by ghickman**
This tweaks so Project page bits so we conform to the expectations of #1379.  I've also used it to capture some other little bugs I found along the way.

### CODE REVIEW

File: opentech/apply/projects/templates/application_projects/project_detail.html [hunk 1]
```diff
@@ -166,11 +166,26 @@ <h4 class="modal__header-bar">Request Payment</h4>
...
+                    <p><a href="{{ object.contracts.first.file.url }}">Draft contract</a></p>
+                </div>
+                {% endif %}
+
                 {% if request.user.is_apply_staff and approvals %}
                 {% block project_approvals %}{% endblock %}
                 {% endif %}
 
-                {% if contracts %}
+                {% if not object.is_in_progress and contracts %}
```
Comment (todd-dembrey) [inline opentech/apply/projects/templates/application_projects/project_detail.html]: We should be showing the contract when they exist, they are relevant at all times.

File: opentech/apply/projects/templatetags/contract_tools.py [hunk 2]
```diff
@@ -10,6 +10,9 @@ def user_can_upload_contract(project, user):
     if user.is_apply_staff:
         return project.status != COMMITTED
 
+    if project.status != CONTRACTING:
```
Comment (todd-dembrey) [inline opentech/apply/projects/templatetags/contract_tools.py]: Why did you add this? If there is a contract to sign they should be able to upload, which the following logic covers.

File: opentech/apply/projects/templates/application_projects/project_detail.html [hunk 3]
```diff
@@ -166,11 +166,26 @@ <h4 class="modal__header-bar">Request Payment</h4>
...
+                    <h5>Supporting Documents</h5>
+
+                    <p><a href="{{ object.submission.get_absolute_url }}">Proposal</a></p>
+
+                    {% if request.user.is_apply_staff %}
+                    <p><a href="#">Approval form</a></p>
+                    <p><a href="#">Supporting documents</a></p>
+                    {% endif %}
+
+                    <p><a href="{{ object.contracts.first.file.url }}">Draft contract</a></p>
```
[the remark this answers is not in the document]
Comment (ghickman) [reply opentech/apply/projects/templates/application_projects/project_detail.html]: > Template updating to reflect the design (contracts subheading within supporting docs sidebar)

As per the spec (Page Variations -> In Progress -> Design (opens Page 10)) the designs show "Draft contract" in the sidebar.

Could we do a better job at finding a draft contract? In hindsight, yes, but this is here because the ticket and design say it should be, if either of those are incorrect then we should fix those.

File: opentech/apply/projects/templatetags/contract_tools.py (hunk 2 above)
Comment (ghickman) [reply opentech/apply/projects/templatetags/contract_tools.py]: Poor assumption on my part, we've gated access to other things after moving out of a relevant state and I thought that applied here too since the spec didn't call it out one way or another.

Removed and updated tests to reflect the correct flow.

File: opentech/apply/projects/templates/application_projects/project_detail.html (hunk 1 above)
Comment (ghickman) [reply opentech/apply/projects/templates/application_projects/project_detail.html]: Ah, I misunderstood this line in the ticket:

> Applicant can only see proposal and contracts

Fixed and rebased.

**Comment by ghickman** [comment]
> You also miss off removing the supporting documents block from the detail page when in progress.

Thanks, I must have removed the check from the model at some point!  Fixed and rebased.

File: opentech/apply/projects/templates/application_projects/project_detail.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (todd-dembrey) [reply opentech/apply/projects/templates/application_projects/project_detail.html]: lets remove it - we've promoted contracts to its own block and are showing that at all times so the designs are no longer relevant.

**Comment by todd-dembrey** [comment]
`{% include "application_projects/includes/supporting_documents.html" %}`
still showing here with no surrounding logic:
https://github.com/OpenTechFund/opentech.fund/blob/4048ec371c8c9c4a297da8aa7619078faae3d8a4/opentech/apply/projects/templates/application_projects/project_detail.html#L118

File: opentech/apply/projects/templates/application_projects/project_detail.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (ghickman) [reply opentech/apply/projects/templates/application_projects/project_detail.html]: Fixed in 09d602c3

**Comment by todd-dembrey** [comment]
> `{% include "application_projects/includes/supporting_documents.html" %}`
> still showing here with no surrounding logic:
> https://github.com/OpenTechFund/opentech.fund/blob/4048ec371c8c9c4a297da8aa7619078faae3d8a4/opentech/apply/projects/templates/application_projects/project_detail.html#L118

^^ still unaddressed.

**Comment by ghickman** [comment]
> > `{% include "application_projects/includes/supporting_documents.html" %}`
> > still showing here with no surrounding logic:
> > https://github.com/OpenTechFund/opentech.fund/blob/4048ec371c8c9c4a297da8aa7619078faae3d8a4/opentech/apply/projects/templates/application_projects/project_detail.html#L118
> 
> ^^ still unaddressed.

Oops, missed that one, thanks.  Fixed in acd2b0d9
